scripts/setup-vitest.js

```diff
@@ -25,14 +25,10 @@ apis.forEach(api => {
 });
 
 beforeAll(() => {
-    // jsdom doesn't come with a WebCrypto implementation (required for uuid)
-    global.crypto = {
-        getRandomValues: function (buffer) {
-            return randomFillSync(buffer);
-        }
-    };
```

Comment:
No longer needed anymore since Web API crypto is now built in

Reply:
yay